liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/MavenLog.java

```diff
@@ -0,0 +1,152 @@
+package org.liquibase.maven.plugins;
+
+import org.apache.maven.plugin.logging.Log;
+
+public class MavenLog implements Log {
+
+    public enum Level {
+        DEBUG(0),
+        INFO(1),
+        WARNING(2),
+        ERROR(3),
+        ;
+
+        private final int level;
+
+        Level(int level) {
+            this.level = level;
+        }
+
+        public boolean isAtLeast(Level other) {
+            return this.level <= other.level;
+        }
+    }
```

Comment:
Is there a reason this is using a duplicate implementation of what is available in `java.util.logging.Level`?

Reply:
Yes:

- maven only supports the 4 log levels, and there is no `DEBUG` and `ERROR` in `java.util.logging.Level`
- The `isAtLeast` (or something equivalent) method also does not exist.

It would be possible to use `java.util.logging.Level`, but then `ERROR` would have to be renamed to `SEVERE` and `DEBUG` to `FINE.*`. This would be less user friendly, since maven uses `debug` and `error` names. Additionally, all the other levels would have to be filtered out. So, using `java.util.logging.Level` would result in even more code.

Reply:
Thanks for the explanation, looks like you made the right decision here.